frontend/package.json

```diff
@@ -29,5 +29,6 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
nit: does the pinning with the hash work across all operating systems? For Python dependencies, it is not uncommon that dependencies have different hashes depending on the OS. But not sure how it works exactly in this case.

Reply:
Great question! It should be the same across all OSes.

In regards to moving to pnpm etc, this will also make the transition easier because when we're ready to switch, the value here in `packageManager` will change to which ever version of pnpm we land on. That will then enforce all users running commands to be on the correct package manager, which they can more easily do so by running `corepack install`.